### DISCUSSION TITLE
2024-10-11 - SSW Sales Pro - Initial meeting GPT - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#9389] [Chewing the Fat]
https://www.ssw.com.au/rules/prepare-for-initial-meetings

Ideas - Thinking about initial meetings and sales in general, what GPTs would be amazing?
(Or already exist)

**Comment by leahy268** [comment]
>Ideas - Thinking about initial meetings and sales in general, what GPTs would be amazing?

SSW Sales Pro looks great.  Would be amazing if we could capture everything from a spec review and use that to generate the spec review doc.  Maybe that's a bit hard because the capture would sometimes be very long.

**Comment by TobyChurches** [comment]
> Thinking about initial meetings and sales in general, what GPTs would be amazing?

@Levijj22's new GPT is a game changer! this would definitely help when you are time poor and need to prepare. From what I saw when I first used it, it seems that the company briefing was very general (this could be due to a bad example I used). It would be good to have a bit more detail about the company itself - potentially other products/services they provide for better context

**Comment by ozdemir-mehmet** [comment]
It looks good for preparing a list of questions if it's an enquiry you're unfamiliar with.

**Comment by ricksu978** [comment]
> Ideas - Thinking about initial meetings and sales in general, what GPTs would be amazing?

If recording the initial meeting is permitted, NotebookLM could be a valuable tool. It can assist with summarizing the content, and even convert it into a podcast, making it easier to revisit and refresh your memory later.

**Comment by jakebayliss** [comment]
Looks cool - will use it for the next IM

**Comment by drwharris** [comment]
>Ideas - Thinking about initial meetings and sales in general, what GPTs would be amazing?

For me, I want to know as much about the business and what they do as possible. Then I want to know about the industry as a whole. I want to be able to talk to t he client on their level.

**Comment by StefStar** [comment]
Super detailed and interesting, can help uncover ideas about an industry you have no familiarity with.

**Comment by joshbermanssw** [comment]
Looks great, optimising with AI is always a benefit. The GPT seems like a great tool to cure the nerves for an initial meeting!

**Comment by calumjs** [comment]
Suggestions for extra features:
- Look up CRM to get all emails/previous projects with client for extra context
- Look up CRM to suggest devs based on skills and availabilities
- Look up similar projects with other clients / case study videos etc
- Do a retro based on the transcript to improve for next time

**Reply by AttackOnMorty** [reply]
Love the last point

**Comment by Ben0189** [comment]
Great Video, i like the Sales GPT, will use it for my next IM

**Comment by kikibianc** [comment]
YakShaver + SSW Sales Pro?
Wouldn't YakShaver be able to do something very similar?

**Comment by matt-goldman-ssw** [comment]
Neat tool, could be useful

**Comment by PennyWalker** [comment]
Great demo! I think that GPT will really alleviate the stress for anyone starting out. It's the most common thing I see as an account manager—the devs always want to know how much to engage and what questions to ask. I think this will help them feel confident in their first meeting, which is key to customer confidence. Great stuff! 

For account managers, it's a time saver! We often jump on enquires the same day and it will give us a preparedness advantage and save a lot of time drafting emails.

**Comment by AntPolkanov** [comment]
> Thinking about initial meetings and sales in general, what GPTs would be amazing?

A GPT which would do client research and provide the main stats:
- number of employees
- goals
- values
- business model
- operational structure
- market analysis.

**Comment by miichaelsmedley** [comment]
Preparation is key to successful meetings, and this tool is a great aid for that. One thing to keep in mind is that sales, especially in our field, are about 98% based on emotional connection. When selling our services, it's crucial to convey competence and make the client feel heard. It's about showing that both you and the developer are invested enough to understand their needs.

So, my advice is to approach meetings with calm confidence. Use your "late-night DJ voice"—be steady, composed, and authoritative, especially when discussing technical subjects. Try to grasp the underlying issue behind what the client is saying. For example, they might ask for an app to automate processes, but their real goal could be to reduce costs or simplify complexity. The better you understand their true objectives, the more effectively you can connect and offer solutions.

**Reply by SamBWagner** [reply]
#WISDOM #MIKEFORPRESIDENT

**Comment by uly1** [comment]
AI FTW!!! Next step... fully autonomous sales avatar agents.

**Comment by isaaclombardssw** [comment]
The AI tool is great – saves time for people who do these kind of meetings a lot, and for a newbie (like me) at this kinda stuff it gives a lot of context to what goes on.

Really sweet. 👍 👍

**Comment by amankumarrr** [comment]
I believe we should compile use cases from all our previous clients, including their initial inquiries and final deliveries, which may be a time-consuming task initially. Once gathered, we can feed them into an AI system. When a new lead comes in, the trained model could analyze the client’s requirements and translate them into meaningful insights. This would not only assist the developers but also benefit the client by showcasing similar successful projects we’ve delivered. It would make a strong first impression by providing relevant case studies in response to their inquiry. Since clients often struggle to articulate their needs clearly, this approach would help bridge that gap effectively.

**Comment by bahjatalaadelssw** [comment]
Great for a boiler plate and for checking any overlooked details.

**Comment by RobJT-wq** [comment]
I think it's a useful tool that I'd use in the future.

**Comment by tkapa** [comment]
One of the hardest parts about initial meetings for me is coming up with those questions to ask. If this can help, then that could be really cool.

**Comment by jeoffreyfischer** [comment]
Love this tool - I will definitely use it for my first Initial Meeting :)

**Comment by steven0x51** [comment]
Great video, SSW Sales Pro is very cool. Will use it next IM

**Comment by ncn-ssw** [comment]
SSW Sales Pro certainly looks like a useful tool for covering all angles and educating less-experienced members.

As GPTs get better, might be able to have one that generates UI mock-ups during meetings.

**Comment by SamBWagner** [comment]
> Ideas - Thinking about initial meetings and sales in general, what GPTs would be amazing?

Not really. I like this one though. :) I'll give it a red-hot crack on the next initial meeting :)

**Comment by chrisschultzssw** [comment]
Nice one! I'll definitely use this for initial meetings in the future.

**Comment by Tanya-Leahy** [comment]
SSW Sales Pro looks like a great tool. I really like Anton's idea for a GPT as well. I can see how this would be helpful in gaining beneficial information/stats about the client's company prior to the IM.

**Comment by sam-s-maher** [comment]
On top of sparking the right questions to ask, generating documents that are consistent between SSW meetings and spec reviews would help to align and sharpen our practices. An all around benefit for onboarding someone like me new to the consulting game.

**Comment by Marxoz** [comment]
- Spec Review helper (with notes etc)
- Tender requirements analysis and response helper

**Comment by JackDevAU** [comment]
An amazing video team! Will try it out on my next Initial meeting

**Comment by awaltos** [comment]
AI is a good tool to help formulate some initial questions that are targeted to the company. Being able to adapt to responses in a client meeting is something that requires experience and not so easily replaced by AI. I feel that GPTs should be used in addition to other prep work - as @miichaelsmedley mentioned, sales meetings involve emotion and are more than just asking q's written by AI. A GPT that gives insight into the people you are about to meet with would also be helpful.

**Comment by stephanfakossw** [comment]
This looks like it could be very useful!

**Comment by duanxinhuan** [comment]
I think it can be even a scrawler a people who ask question, like scraw client website client employee etc…
so when we have initial meeting we know them fully. But I love the idea, anything help getting client is something I have genius love for because I want ssw to flourish.

**Comment by babakamyljanovssw** [comment]
Great demo, I will use it for my first Initial Meeting

**Comment by Aibono1225** [comment]
The tool looks really useful! I'll definetely try it out for my first initial meeting

**Comment by jimmidier** [comment]
Great video. I will definitely try on my next initial meeting

**Comment by jerryluossw** [comment]
It's awesome, is it possible to integrate this into Teams or other SSW Products, like RulesGPT or SophieBot? so we can use it conveniently in China, in addition, we can integrate more GPTs in the future.

**Comment by tino-liu** [comment]
Oh the AI looks like a very useful tool! Will definitely try it for the next initial meeting

**Comment by Calinator444** [comment]
> Ideas - Thinking about initial meetings and sales in general, what GPTs would be amazing?
(Or already exist)

Maybe a GPT that analyzes potential business requirements the company may need in the future. If you can solve problems within the organization you may secure potential work in the future. This is getting into the territory of business analysis though.

**Comment by MattParkerDev** [comment]
Sales Pro GPT looks very interesting, will definitely give it a try on my next initial meeting.

**Comment by bettybondoc** [comment]
Looks great. Haven't done much initial meetings but this tool looks useful.

**Comment by vladislav-kir** [comment]
> Ideas - Thinking about initial meetings and sales in general, what GPTs would be amazing?

Sales Pro GPT looks promising! Integrating it with CRM could bring some extra insight, for example developers suitable for the projects, or details of previous projects if this is an existing client

**Comment by ozair-ashfaque** [comment]
It is a great video and tool, and I will use it for my first initial meeting with the client.

AI can help answer initial questions for a company, but responding on the spot in client meetings is something that takes experience.

**Comment by danielmackay** [comment]
One thing I've seen @miichaelsmedley do is to record a transcript of the meeting (after confirming the client is fine with that).  This is useful to feed into ChatGPT to summarise the meeting, and help build confidence with the client and prove that we have not missed anything.
